src/main/java/com/icthh/xm/tmf/ms/resource/config/WebMvcConfiguration.java

```diff
@@ -4,21 +4,22 @@
 import com.icthh.xm.commons.web.spring.TenantInterceptor;
 import com.icthh.xm.commons.web.spring.XmLoggingInterceptor;
 import com.icthh.xm.commons.web.spring.config.XmWebMvcConfigurerAdapter;
-import java.util.List;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
 
+import java.util.List;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version